app/src/main/java/com/kickstarter/ui/activities/CreatePasswordActivity.kt

```diff
@@ -62,6 +63,8 @@ class CreatePasswordActivity : AppCompatActivity() {
                         AppThemes.LIGHT.ordinal -> false
                         else -> false
                     }
+                } else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
[not mandatory], but we could try to move all theme related logic to the composable `KickstarterApp`, trying to avoid having to repeat in every single screen the theming setup.

Reply:
thats a good idea, just not sure how much access the kickstarter app has to builds and stuff, but ill play around with that, would save a lot of time